README.rst

```diff
@@ -0,0 +1,80 @@
+Welcome to shimming-toolbox!
+============================
+
+|badge-ci| |badge-coveralls| |badge-doc| |badge-twitter|
+
+.. |badge-ci| image:: https://github.com/shimming-toolbox/shimming-toolbox/workflows/CI-Tests/badge.svg?
+    :alt: GitHub Actions CI
+    :target: https://github.com/shimming-toolbox/shimming-toolbox/actions?query=workflow%3ACI-Tests+branch%3Amaster
+
+.. |badge-coveralls| image:: https://coveralls.io/repos/github/shimming-toolbox/shimming-toolbox/badge.svg?branch=master
+    :alt: Coverage Status
+    :target: https://coveralls.io/github/shimming-toolbox/shimming-toolbox?branch=master
+
+.. |badge-doc| image:: https://readthedocs.org/projects/shimming-toolbox-py/badge/?version=latest
+    :alt: Documentation Status
+    :target: https://shimming-toolbox.org/en/latest/
+
+.. |badge-twitter| image:: https://img.shields.io/twitter/follow/shimmingtoolbox.svg?style=social&label=Follow
+    :alt: Twitter Follow
+    :target: https://twitter.com/shimmingtoolbox
+
+``shimming-toolbox`` is an open-source Python software package enabling
+a variety of MRI shimming (magnetic field homogenization) techniques
+such as
+`static <https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.25587>`__
+and `real-time <https://doi.org/10.1002/mrm.27089>`__ shimming for use
+with standard manufacturer-supplied gradient/shim coils or with custom
+"multi-coil" arrays. The toolbox provides useful set of command line tools as
+well as a `fsleyes plugin <https://github.com/shimming-toolbox/fsleyes-plugin-shimming-toolbox#fsleyes-plugin-for-shimming-toolbox>`__
+dedicated to make shimming more accessible and more reproducible.
+
+Insert image of the overview
+
+Features
+________
+
+* Built-in DICOM to NIFTI conversion
+* Supports masking of spinal cord
+* Supports the usage and creation of custom coil profiles
+* Supports different shimming approaches: spherical harmonics, multi-coil, gradient shimming, dynamic, realtime
+* Built using freely available software tools: `SCT <https://spinalcordtoolbox.com/en/latest/>`__, `FSL-prelude <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FUGUE/Guide#PRELUDE_.28phase_unwrapping.29>`__, `dcm2niix <https://github.com/rordenlab/dcm2niix>`__, `dcm2bids <https://github.com/UNFmontreal/Dcm2Bids>`__
+
+Installation
+____________
+
```

Comment:
I see why having this heading here helps with the overview, but on the side toolbar it looks like we have two installation sections - not sure if this is confusing? What do other people think?

Reply:
we just did a small poll and i think we prefer it the way it is now